### PR TITLE
Increase AsyncSnapshotWriterImplTest timeout

### PR DESCRIPTION
**Pull request by viliam-durina**
Maybe fixes https://github.com/hazelcast/hazelcast-jet/issues/2972. The logs are garbled (it's a parallel test), the full
log is garbled too, artifacts have 45 bytes:
https://s3.console.aws.amazon.com/s3/object/j-artifacts?region=us-east-1&prefix=jet-oss-maintenance-oracle-jdk8/261/jet-oss-maintenance-oracle-jdk8-261.tar.
So this is just an attempt.

### CODE REVIEW

**Comment by viliam-durina** [comment]
run-lab-run

**Comment by viliam-durina** [comment]
run-lab-run

File: hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/validate/types/HazelcastTypeUtils.java [hunk 1]
```diff
@@ -103,7 +103,8 @@ public static SqlTypeName toCalciteType(QueryDataTypeFamily typeFamily) {
     }
 
     public static QueryDataType toHazelcastType(SqlTypeName sqlTypeName) {
-        switch (sqlTypeName.getFamily()) {
+        // some types have null family (e.g. SqlTypeName.ROW), we fall back to default branch for those
+        switch (sqlTypeName.getFamily() != null ? sqlTypeName.getFamily() : SqlTypeFamily.ANY) {
```
Comment (gierlachg) [inline hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/validate/types/HazelcastTypeUtils.java:107]: That's a separate issue. The other thing is, personally I find #18437 a more readable.

**Review by gierlachg — APPROVED** [review]

**Comment by mmedenjak** [comment]
run-lab-run

**Comment by mmedenjak** [comment]
run-lab-run

File: hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/validate/types/HazelcastTypeUtils.java (hunk 1 above)
Comment (viliam-durina) [reply hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/validate/types/HazelcastTypeUtils.java]: Removed it

**Comment by viliam-durina** [comment]
run-lab-run

**Pull request merged**